src/Avalonia.X11/XShmExtensions/X11ShmDebugLogger.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Avalonia.X11.XShmExtensions;
+
+internal static class X11ShmDebugLogger
```

Comment:
`Logger.TryGet(LogEventLevel.Information, "X11")?.Log(...`

Reply:
@kekekeks Thank you. I intend for the X11ShmDebugLogger to be used by Avalonia framework developers when investigating the XShm module, rather than by application developers using the Avalonia framework. Application developers should not be misled by the debug output from XShm. I'm not sure if this expectation is reasonable.